benchmark/sparse/linear_regression/keras_sparse_model.py

```diff
@@ -0,0 +1,56 @@
+"""
+Linear Regression model with sparse synthetic data for Keras
+"""
+from __future__ import print_function
+
+import time
+
+from keras import Model
+from keras.layers import Dense, Input
+from keras.optimizers import SGD
+from keras import backend as K
+from keras.utils import multi_gpu_model
+
+
+def _validate_backend():
+    if K.backend() != 'mxnet' and K.backend() != 'tensorflow':
+        raise NotImplementedError('This benchmark script only supports MXNet and TensorFlow backend')
+
+
+def run_benchmark(train_data, train_label, eval_data, eval_label, batch_size, epochs, gpus):
+    _validate_backend()
+
+    inputs = Input(batch_shape=(None, train_data.shape[1]), dtype='float32', sparse=True)
+    predictions = Dense(units=1, activation='linear', kernel_initializer='normal')(inputs)
+    model = Model(inputs=inputs, outputs=predictions)
+    model.summary()
+
+    sgd = SGD(lr=0.1, momentum=0.9)
+
+    if gpus > 1:
+        model = multi_gpu_model(model, gpus=gpus)
+
+    model.compile(loss='mse', optimizer=sgd, metrics=['accuracy'])
+
+    model.fit(train_data,
+              train_label,
+              epochs=epochs,
+              batch_size=batch_size, verbose=1)
+
+    start = time.time()
```

Comment:
Let's add an argument `is_train`, or `benchmark_mode`, so users can configure what to benchmark. Time fit for training and only do predict and time for inference.